v22.2/sso-sql.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Cluster Single Sign-On (SSO) for Self-hosted CockroachDB
+summary: Overview of Single Sign-On (SSO) for CockroachDB SQL Access, and review of workflows for authenticating human and bot users, and for configuring the feature.
+toc: true
+docs_area: manage
+---
+
+## What is Cluster Single Sign-On (SSO)?
+
+Cluster SSO allows users to access the SQL interface of a CockroachDB cluster using a JWT auth token issued by a customer-managed identity provider (IdP).
+
+This page discusses use cases for authenticating to {{ site.data.products.core }} clusters. You might instead be looking for [Cluster Single Sign-On (SSO) for Cockroach Cloud](../cockroachcloud/cloud-sso-sql.html) or [Single Sign-On (SSO) for CockroachDB Cloud organizations](../cockroachcloud/cloud-org-sso.html).
+
+**Prerequisites**
+
+- **IdP:**
+
+	You must have the ability to create identities and issue JSON Web Token (JWT) formatted auth tokens.
+
+	This [Cockroach Labs blog post](https://www.cockroachlabs.com/blog/) covers and provides further resources for a variety of token-issuing use cases, including using Okta and Google Cloud Platform to issue tokens.
+
+- **CockroachDB:**
+
+	- You must have access to an enterprise cluster, i.e. configured with a valid [CockroachDB enterprise license](enterprise-licensing.html) enabled.
+
+		See [Enterprise Trial –– Get Started](get-started-with-enterprise-trial.html) for help enabling your cluster with you enterprise license.
+
+	- SQL users/credentials:
+
+		- You must have access to credentials with the [`admin` role](security-reference/authorization.html#admin-role), in order to update cluster settings, which is required to allow an external token issuer/IdP.
+	
+		- A SQL user specifically corresponding to the service account must be pre-provisioned on the cluster (or you must have access to a SQL role allowing you to create such a user).
+
+## Configure your cluster settings
+In order to authenticate a service account to a {{ site.data.products.db }} cluster using a JWT issuer, you must update several cluster settings in the `server.jwt_authentication` namespace:
+
+1. `server.jwt_authentication.enabled`
+
+	JWT authentication must be enabled on your cluster.
+
+	Required value: `true`
+
+1. `server.jwt_authentication.jwks`
+
+	Add your IdP's public signing key to your cluster's list of accepted signing JSON web keys (JWKS), under the `jwks` setting.
+
+	This must be a [JWK](https://www.rfc-editor.org/rfc/rfc7517) formatted single key or key set, containing the [public keys](../{{site.versions["stable"]}}/security-reference/transport-layer-security.html#key-pairs) for SSO token issuers/IdPs that will be accepted by your cluster.
+
+1. `server.jwt_authentication.issuers`
+
+	Add your IdP's formal `issuer` name (this must match the `issuer` field in the JWT itself) to your cluster's list of accepted token issuers:
```

Comment:
This is similar to the cloud page. It seems like the colon could just be a period here. And, the next sentence on the comma-separated list, could just be part of the same paragraph as it's expanding on the last point. (But, maybe I'm not understanding something here.)